### ISSUE TITLE
Source-postgres : Normalization broken on Invalid timestamp: '2021-04-20T00:00'

### ISSUE THREAD

**Issue by ro8inmorgan** [airbytehq/airbyte#13605]
Help, basic normalization doesn't work anymore for me with the latest version 0.39-14 alpha

The error I'm getting is

```
2022-06-08 14:06:27 ERROR c.n.s.DateTimeValidator(isLegalDateTime):70 - Failed to apply RFC3339 pattern on 0001-01-01T00:00
14:06:51.143227 [debug] [Thread-7 ]: BigQuery adapter: Retry attempt 3 of 3 after error: BadRequest("Invalid timestamp: '2021-03-12T00:00'")
2022-06-08 14:06:56 normalization > 14:06:52.921812 [debug] [Thread-7 ]: Database Error in model lever (models/generated/airbyte_tables/airbytetest/lever.sql)
2022-06-08 14:06:56 normalization > Invalid timestamp: '2021-04-20T00:00'
2022-06-08 14:06:56 normalization > compiled SQL at ../build/run/airbyte_utils/models/generated/airbyte_tables/airbytetest/lever.sql
2022-06-08 14:06:56 normalization > 14:06:52.922293 [error] [Thread-7 ]: 3 of 3 ERROR creating table model airbytetest.lever..................................................................... [ERROR in 5.29s]

```
Source Postgress, Destination BigQuery

**Comment by ro8inmorgan** [comment]
So further digging the field should not be set to DATETIME based on the content of '2021-04-20T00:00' but because it is set to DATETIME the generated normalization by Airbyte fails because BigQuery cannot cast this format to DATETIME

**Comment by ro8inmorgan** [comment]
Ok so the bug seems with the latest postgress source connector 4.21, switching back to 4.16 solved it for me

**Comment by grishick** [comment]
This should be fixed in version `0.4.24`
